Emby.Server.Implementations/LiveTv/EmbyTV/EncodedRecorder.cs

```diff
@@ -303,19 +304,14 @@ private void OnFfMpegProcessExited(Process process)
 
         private async Task StartStreamingLog(Stream source, Stream target)
         {
+            byte[] buffer = ArrayPool<byte>.Shared.Rent(IODefaults.CopyToBufferSize);
             try
             {
-                using (var reader = new StreamReader(source))
+                int bytesRead;
+                while ((bytesRead = await source.ReadAsync(new Memory<byte>(buffer)).ConfigureAwait(false)) != 0)
```

Comment:
Will it create a new `Memory<byte>` on every iteration? And is it necessary?

Reply:
It will, but `Memory` is a struct, so I don't think it matters

Reply:
It's still allocated on the stack, isn't it? It doesn't matter because it's optimized by the compiler?